### DISCUSSION TITLE
Empty Launch While Uploading Results on Allure TestOps

### DISCUSSION THREAD

**Discussion by 75andybermond** [allure-framework/allure2#2033] [TestOps]
Issue Summary:

When uploading result with AllureCTL tool, the initial report is generated successfully. However, when attempting to update the results on the launch panel, it appears to be empty without any data. This issue occurs when using the command manually and also during the CI process, where it gets stuck before sending anything on 'watcher wait.'

Steps to Reproduce:

    Run a test plan that generates reports
    Run the command to generate the Allure report manually or as part of the CI process.
    Open the Allure TestsOps UI and look at the launch results.
    Observe the corresponding launch is displayed correctly.
    Notice that the launch appears empty (0 test executed), with no test results displayed.

![image](https://github.com/allure-framework/allure2/assets/107470329/4bde2593-231e-4f33-a04f-498f3a44883c)

Additional Information:

    Operating System: Ubuntu 20.
    Browser/Environment: Chrome
    Command used to generate the report: 
```

       curl -L0 https://github.com/allure-framework/allurectl/releases/latest/download/allurectl_linux_386 -o ./allurectl
        chmod +x ./allurectl
	make qa_pytest_classic || touch test_failed
	./allurectl upload --endpoint https://allure.xxx.lan/ \
	--token xxxx-xxxx-xxxx\
	--project-id 4 \
	--launch-name "$$(git log --pretty="%h %s" -1)" \
	--launch-tags "Integ, Dev, $$(id -un)" \
	qa/reports
```

    Relevant configuration or code snippets: 
    Any error messages or logs: `watchers wait`

Possible Workarounds:

    Manually delete the existing report and generate a new one.
    Use an alternative reporting tool or framework temporarily.

**Comment by cheshi-mantu** [comment]
@75andybermond Hi Andy,
thanks for contacting us. We'll try to help you here, but just for the future reference Allure Testops is a commercial product having its own dedicated help portal here: https://help.qameta.io (you need to register using your company's email).
You provided the way you are trying to upload the data located in `qa/reports`, but what exactly is the data – is it raw allure results or the report generated? Allure Testops only processes results files (*-result.json, *-container.json, *-attachment.[ext]), not the generated report files.
